index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+  <meta charset="utf-8">
+  <title>Vboo's Car Blog!</title>
+  <script type="text/javascript" src="app.js"></script>
+  <link rel="stylesheet" href="index.css"/> 
+</head>
+
+<header class="header">
+    <h1>Vboo's Car Blog </h1> 
+</header>
+
+<body>
+  <main>
+  <div class="intro-box">
+    <p>Welcome and thanks for stopping by my car blog! I will post some of my favorite cars and information regarding their specs!</p>
+    <img id="Home_Picture" src="https://hips.hearstapps.com/hmg-prod/amv-prod-cad-assets/images/16q3/669463/lightning-lap-2016-the-years-hottest-performance-cars-at-vir-feature-car-and-driver-photo-670701-s-original.jpg?resize=480:*" </img>
+  </div>
+  <div>
+    <h1>Cars</h1>
+    <h1>BMW M5</h1>

```

Comment:
Add links to your site so it's easier to navigate between pages.

<a href="carOne.html"><h1>BMW M5</h1> </a> should do the trick